### PR TITLE
installs newrelic

### PR DESCRIPTION
**Pull request by lsh-0**
(did you know elife-dashboard is still on python 2 ??)

### CODE REVIEW

**Comment by giorgiosironi** [comment]
(did you know elife-dashboard is still on python 2 ??)

File: install.sh [hunk 1]
```diff
@@ -5,3 +5,4 @@ if [ ! -d venv ]; then
 fi
 source venv/bin/activate
 pip install -r requirements.txt
+NEW_RELIC_EXTENSIONS=false pip install --no-binary :all: newrelic==2.82.0.62
```
Comment (giorgiosironi) [inline install.sh]: it's also a very old version, required by Python 2?

Comment (lsh-0) [reply install.sh]: copied from lax. we should definitely look at upgrading it, I'll open a ticket with the affected projects

**Comment by lsh-0** [comment]
it's funny you post that cat, I have two cats that hang around my place and one looks just like this one. Him and a black+white cat were having a street fight just minutes ago outside my office window. I had to break them up with a broom. Fur *everywhere*.

**Comment by giorgiosironi** [comment]
Yet another predator animal destabilizing Australia's ecosystem :cat:

**Comment by lsh-0** [comment]
yeah, you're absolutely right. I have a tree full of native birds right outside my office window and the cat that resembles the one in the phone will climb the tree and lie on the trunk (well camouflaged) and stalk them. Never seen him catch one, he's a bit fat for that, but he definitely has a murderous hobby.

**Pull request merged**